build/stf-run-ci/tasks/create_catalog.yml

```diff
@@ -151,7 +185,25 @@
       dest: "{{ base_dir }}/working/service-telemetry-framework-index/index.yaml"
 
   - name: Build service-telemetry-framework-index
-    ansible.builtin.command: oc start-build -n "{{ namespace }}" service-telemetry-framework-index --wait --from-dir {{ base_dir }}/working/service-telemetry-framework-index
+    ansible.builtin.command: oc start-build -n "{{ namespace }}" service-telemetry-framework-index --from-dir {{ base_dir }}/working/service-telemetry-framework-index
+    ignore_errors: true
+
+  - name: Kill the first build
+    ansible.builtin.shell:
+      oc delete build service-telemetry-framework-index-1 -n {{ namespace }}
+    ignore_errors: true
+    retries: 3
+    delay: 10
+    register: kill_build
+    until: kill_build.rc == 0
+
+  - name: Build service-telemetry-framework-index again
+    ansible.builtin.command: oc start-build -n "{{ namespace }}" service-telemetry-framework-index --wait --follow --from-dir {{ base_dir }}/working/service-telemetry-framework-index
```

Comment:
This command is different from the one above. I think you can probably just remove the 2 commands above this?